Sources/Quick/QuickSpec.swift

```diff
@@ -3,26 +3,26 @@ import XCTest
 // NOTE: This file is not intended to be included in the Xcode project or CocoaPods.
 //       It is picked up by the Swift Package Manager during its build process.
 
-public class QuickSpec: XCTestCase, XCTestCaseProvider {
+public class QuickSpec: XCTestCase {
     public func spec() {}
 
     public required init() {}
```

Comment:
As I tested, this implementation is not compatible with `swift test` on OS X.
`XCTestCase` on OS X has `required init()`, so we need to use `override` and add calling `super.init()`.
`super.init()` calls `init(invocation:)` inside of that and generate error:
> …/Sources/Quick/QuickSpec.swift: 8: 14: fatal error: use of unimplemented initializer 'init(invocation:)' for class 'Quick.QuickSpec'